types/ckeditor__ckeditor5-engine/src/model/selection.d.ts

```diff
@@ -27,6 +27,7 @@ export type Selectable =
     | Element
     | RootElement
     | Text
+    | null
```

Comment:
I think it's a bit odd for the `null` to be part of this union (rather than to type the methods as `setSelection(selectable?: Selectable | null)`), but I also cannot find a single place where `Selectable` is used where null isn't allowed, so I suppose it makes sense.